src/App.tsx

```diff
@@ -10,6 +10,7 @@ import { Outlet } from "react-router-dom"
 
 import getClient from "./Client"
 import * as consts from "./Consts"
+import {getUser, removeUser} from "./UserLogin"
```

Comment:
`getUser()` is unused now. The linter or compiler is probably going to fail on that.

Reply:
Removed it in the latest commit. Will push in a minute